ams/dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -28,6 +28,8 @@ ams:
 
   self-optimizing:
     commit-thread-count: 10
+    runtime-data-keep-days: 30
+    runtime-data-expire-interval-hours: 1
```

Comment:
I saw data-expiration.interval is configure (`1d`) and I thought we should use the same style.

Reply:
+1

Reply:
Using a configuration value with units indeed provides more flexibility, but considering the format of some historical configurations, perhaps we can support this new configuration method in another PR to ensure uniformity.

Reply:
It would be a good idea to unify the time units in another PR, and also sort out the historical configuration.